resources/lang/en/community.php

```diff
@@ -126,25 +126,4 @@
             'instructions' => 'click the heart button to proceed to the osu!store',
         ],
     ],
-    'slack' => [
```

Comment:
As far as I know there will be a new page made for public discord anyways, but these changes are available to view and copy if we need them